.github/workflows/continuous-integration.yaml

```diff
@@ -54,8 +54,8 @@ jobs:
       run: ./cd/lint_charts.sh
 
     - name: Deploy to Kubernetes
-      run: ./cd/deploy_to_kind.sh
-  
+      run: ./cd/deploy_to_kind.sh ./docker/accumulo2.env
```

Comment:
This is just a tidy up for the environment file instead of using the Jupyter one - as changed in `cd/update_versions.sh`?

Reply:
It is, it just makes it so there is one source of truth for the versions so not duplicated everywhere.